### PR TITLE
Fix WC old versions compatibility

### PR DESCRIPTION
**Pull request by Abdalsalaam**
### Description
https://wordpress.org/support/topic/fatal-error-since-update-3-4-6/

### CODE REVIEW

**Review by iyut — CHANGES_REQUESTED** [review]
Hi @Abdalsalaam , I made some comments on this PR

File: pr-dhl-woocommerce/includes/class-pr-dhl-wc-order-paket.php [hunk 1]
```diff
@@ -955,13 +953,7 @@ public function add_order_label_column_header( $columns ) {
 	public function add_order_label_column_content( $column, $order = null ) {
 		global $post;
```
[the remark this answers is not in the document]
Comment (Abdalsalaam) [reply pr-dhl-woocommerce/includes/class-pr-dhl-wc-order-paket.php:954]: @iyut order parameter will be passed only if HPOS enabled, isn't it ?

[the remark this answers is not in the document]
Comment (iyut) [reply pr-dhl-woocommerce/includes/class-pr-dhl-wc-order-paket.php:954]: based on [this line](https://github.com/WordPress/WordPress/blob/d097367dbbbf1d0ffc584202b80e9136bd324a0e/wp-admin/includes/class-wp-posts-list-table.php#L1387), it's actually post ID.

[the remark this answers is not in the document]
Comment (iyut) [reply pr-dhl-woocommerce/includes/class-pr-dhl-wc-order-paket.php:954]: and for HPOS, we will need to use this action : `add_action( 'manage_woocommerce_page_wc-orders_custom_column', array( $this, 'add_order_label_column_header' ), 1, 2 );`

[the remark this answers is not in the document]
Comment (iyut) [reply pr-dhl-woocommerce/includes/class-pr-dhl-wc-order-paket.php:954]: This is the snippet that I use for checking the order instance : 

```
add_action( 'manage_shop_order_posts_custom_column', array( $this, 'example_of_custom_column' ), 1, 2 );
add_action( 'manage_woocommerce_page_wc-orders_custom_column', array( $this, 'example_of_custom_column' ), 1, 2 );
public function example_of_custom_column( $column, $post_id_or_order ) {
		$order = ( $post_id_or_order instanceof WC_Order ) ? $post_id_or_order : wc_get_order( $post_id_or_order );

		if ( ! ( $order instanceof WC_Order ) ) {
			return;
		}
...
...
...
}
```

Hope this make sense.

**Review by iyut — APPROVED** [review]
The changes LGTM!

**Pull request merged**